### PR TITLE
feat: support HTTPS in webui

### PR DESCRIPTION
**Pull request by patriciareinoso**
This PR enables the use of HTTPS for the webui service.

The TLS certificate and private key paths are parametrized on the configuration file.

```
configuration:
   tls:
      key: /support/TLS/webui.key # Webui TLS private key
      pem: /support/TLS/webui.pem # Webui TLS Certificate
```

### CODE REVIEW

**Review by gruyaume — APPROVED** [review]

File: backend/webui_service/webui_init.go [hunk 1]
```diff
@@ -190,19 +190,22 @@ func (webui *WEBUI) Start() {
 	go func() {
 		httpAddr := ":" + strconv.Itoa(factory.WebUIConfig.Configuration.CfgPort)
 		logger.InitLog.Infoln("Webui HTTP addr", httpAddr)
-		if factory.WebUIConfig.Info.HttpVersion == 2 {
+		tlsConfig := factory.WebUIConfig.Configuration.TLS
+		if factory.WebUIConfig.Info.HttpVersion == 2 || tlsConfig != nil {
```
Comment (gab-arrobo) [inline backend/webui_service/webui_init.go:194]: Why do you need this `|| tlsConfig != nil`? My understanding is that we are interested in Http version 2.0, which can be either insecure (http => `ListenAndServe`) or secure (https => `ListenAndServeTLS`), and this is checked by `if tlsConfig != nil` in line 204. However, by having `|| tlsConfig != nil` here, we are saying that the Http version can be <= 2. Is this the intention of this change?
```suggestion
		if factory.WebUIConfig.Info.HttpVersion == 2 {
```

**Comment by gatici** [comment]
Please rebase this branch.

File: backend/webui_service/webui_init.go (hunk 1 above)
Comment (gatici) [reply backend/webui_service/webui_init.go:194]: Agree.

**Comment by gatici** [comment]
I suggest to update the docs.
In api_default.go:
// @Router       /config/v1/network-slice/{slice-name}  [get]   L198
// @Router      /config/v1/network-slice/{slice-name}  [delete] L221
 // @Router       /config/v1/network-slice/{slice-name  [post]   L243
 
 In docs.go:
 
"/config/v1/network-slice/{slice-name}": {  L852

**Review by gab-arrobo — APPROVED** [review]
+1

**Pull request merged**